### DISCUSSION TITLE
joomla 5 compatability

### DISCUSSION THREAD

**Discussion by raramuridesign** [jreviews/plg_loginwithemail#3] [Q&A] [answered]
Hi Team
Does this work on J5... 
M.

**Comment by jreviews** [comment]
@raramuridesign The code is very straightforward and I don't see any reason why it wouldn't be compatible. Take it for a spin and let us know.

**Comment by raramuridesign** [comment, marked as answer]
I can confirm its working @jreviews 

I had one question though. Would it be possible to force the user to use email rather than username ?
M

**Reply by jreviews** [reply]
@raramuridesign That's beyond the scope of the plugin which only acts as a middleman between the form submission and Joomla's actual processing of the form. My suggestion for a quick solution would be to change the login input for the username to type="email" which would give you browser-only validation.

**Reply by raramuridesign** [reply]
@jreviews Thank you. We will give this a go.
